### PR TITLE
refactor(ui): add latest labels for Italian language localization

### PR DESCRIPTION
**Pull request by polilluminato**
Italian translation

### CODE REVIEW

**Comment by russellwheatley** [comment]
Hey @polilluminato, do you mind fixing the formatting and analyse CI issues so we can get this merged in, please?

File: packages/flutterfire_ui/lib/src/i10n/lang/it.dart [hunk 1]
```diff
@@ -251,14 +251,14 @@ class ItLocalizations extends FlutterFireUILocalizationLabels {
...
     this.emailVerificationInstructionText =
-        'A verification email has been sent to your email address. '
-            'Please check your email and click on the link to verify '
-            'your email address.',
-    this.couldntVerifyEmailText = "We couldn't verify your email address. ",
-    this.resendVerificationEmailButtonLabel = 'Resend verification email',
+        'Una email di verifica è stata appena inviata al tuo indirizzo email. '
+            'Controlla la tua email e clicca sul link per la verifica '
+            'del tuo indirizzo email.',
+    this.couldntVerifyEmailText = "Non possiamo verificare il tuo indirizzo email. ",
```
Comment (Salakar) [inline packages/flutterfire_ui/lib/src/i10n/lang/it.dart:260]:
```suggestion
    this.couldntVerifyEmailText = 'Non possiamo verificare il tuo indirizzo email.',
```

**Comment by polilluminato** [comment]
> Hey @polilluminato, do you mind fixing the formatting and analyse CI issues so we can get this merged in, please?

I don't understand with I have to do. I saw that @Salakar already fix the format.

**Comment by russellwheatley** [comment]
Hey @polilluminato, click on details next to the failiing CI tests here:
Screenshot 2022-05-04 at 12 55 42

You can format a file on the command line by going to the directory and running `flutter format .`. You will also need to check the analyze issue and update based on feedback. Thank you.

**Comment by polilluminato** [comment]
@russellwheatley I did it and pushed the code.

**Comment by russellwheatley** [comment]
@polilluminato looks like you'll need to format more files  - `packages/flutterfire_ui/lib/src/i10n/lang` go to this directory and run this command `flutter format .`

Fix these analyse issues:

```bash
[flutterfire_ui]:    info - lib/src/i10n/lang/id.dart:260:35 - Only use double quotes for strings containing single quotes. - prefer_single_quotes
[flutterfire_ui]:    info - lib/src/i10n/lang/ja.dart:254:35 - Only use double quotes for strings containing single quotes. - prefer_single_quotes
```

Thanks!

**Comment by polilluminato** [comment]
@russellwheatley I think I've done all the steps:

- [x] run `flutter format .` in `packages/flutterfire_ui/lib/src/i10n/lang`
- [x] fixed issue on lib/src/i10n/lang/id.dart
- [x] fixed issue on lib/src/i10n/lang/ja.dart

**Comment by russellwheatley** [comment]
If you merge with master, the last analyze issue should disappear, @polilluminato 🙏

**Comment by russellwheatley** [comment]
Something has gone wrong, @polilluminato 😓. PR now changes 466 files...

**Comment by polilluminato** [comment]
> Something has gone wrong, @polilluminato sweat. PR now changes 466 files...

I think I've made a mess. Maybe it's a good idea to dismiss this PR and start from scratch?

**Comment by russellwheatley** [comment]
Hey @polilluminato, if you wish. We'd still gratefully accept another PR submission 😄

**Pull request closed without merging**